go/enclave/nodetype/interfaces.go

```diff
@@ -33,15 +33,6 @@ type ActiveSequencer interface {
 	NodeType
 }
 
-type BackupSequencer interface {
```

Comment:
no longer needed. Because the backup sequencers will be normal validators, now that the mempool has that "ValidateOnly" flag